docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-topology.md

```diff
@@ -158,71 +158,74 @@ To deploy a partition-by-region cluster, refer to [Partition by region](../creat
 - [Tablespaces](../../../explore/ysql-language-features/going-beyond-sql/tablespaces/)
 
 <!--
-## Cross-cluster
+-->
```

Comment:
This is the line I will be removing before landing to comment out the section again